File: app/src/main/java/com/example/msdc/api/MovieResponse.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class MovieResponse {
    @SerializedName("results")
    private final List<MovieResult> results = new ArrayList<>();

    public List<MovieResult> getResult(){ return results; }
}

File: app/src/main/java/com/example/msdc/api/MovieResponse.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class MovieResponse (
    @field:SerializedName("results") val result: ArrayList<MovieResult>? = null
) : Parcelable